opencti-platform/opencti-graphql/src/domain/indicator.js

```diff
@@ -117,7 +117,7 @@ export const addIndicator = async (user, indicator, createObservables = true) =>
   const indicatorToCreate = pipe(
     assoc('main_observable_type', indicator.main_observable_type.toLowerCase()),
     assoc('score', indicator.score ? indicator.score : 50),
-    assoc('valid_from', indicator.valid_from ? indicator.valid_from : Date.now()),
+    assoc('valid_from', indicator.valid_from ? indicator.valid_from : new Date()),
```

Comment:
Function now() from database/grakn should be use instead.

Reply:
Not sure how to do that at this point in the code, previously it was using all the values from Node.

Reply:
You can check addIncident in incident.js
```
import { now } rom '../database/grakn';
const currentDate = now();
```

Reply:
Do you have some time to finish the PR or other question on my remark ? 
If you don't have the time I can modify it a little and merge it. Thanks

Reply:
Because of the method `prepareAttribute` in grakn.js, I was indeed intended to create a new pure Date object in JS. Your PR is working and that's what we need. Thanks a lot!